test_assigment1.py

```diff
@@ -7,3 +7,12 @@ def test_arabicToRoman():
     assert arabicToRoman(3999) == 'MMMCMXCIX'
     assert arabicToRoman(123) == 'CXXIII' 
     assert arabicToRoman(4) == 'IV'
+    assert arabicToRoman(1000) == 'M'
+    assert arabicToRoman(13) == 'XIII'
+    assert arabicToRoman(2414) == 'MMCDXIV'
+    assert arabicToRoman(500) == 'D'
+    assert arabicToRoman(900) == 'CM'
+    assert arabicToRoman(4999) == 'MMMMCMXCIX'

```

Comment:
Can't have more than 3 same letters together - technically not a valid roman number. But that's ok. I regraded your HW1.